### ISSUE TITLE
Core plugin installation utility

### ISSUE THREAD

**Issue by matthyk** [fastify/fastify-cli#695]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

A CLI utility within `fastify-cli`. The process could be similar to the one you go through when you run `npm init`.

### Motivation

If you create a new project using `fastify-cli`, you will normally also use plugins, e.g. for authentication or database access. However, I find it unnecessarily time-consuming to find out the names of the desired plugins, i.e. whether it is now called `fastify-postgres`, `@fastify/postgres` or `@fastify/pg`. Since a list of core plugins is also maintained in the docs, `fastify-cli` could contain a utility for the installation of core plugins. This could also be categorised according to subject areas, e.g. `authentication`, `database access`, `security`, etc.

**Comment by mcollina** [comment]
Would you like to work on this?

I recall there was a UI tool build by @Eomm... I think we could put that on the website.

**Comment by matthyk** [comment]
Absolutely! Do you have any recommendations for a package to build an interactive CLI?

**Comment by mcollina** [comment]
inquirer is the best one.

**Comment by matthyk** [comment]
First draft #696
